src/generator.cs

```diff
@@ -6234,13 +6234,13 @@ public void Generate (Type type)
 							else if (btype == TypeManager.System_UInt64)
 								print ($"return ({fieldTypeName}) Dlfcn.GetUInt64 (Libraries.{library_name}.Handle, \"{fieldAttr.SymbolName}\");");
 							else
-								throw new BindingException (1014, true, "Unsupported type for Fields: {0}", fieldTypeName);
+								throw new BindingException (1014, true, "Unsupported type for Fields: {0} for '{1}'.", fieldTypeName, field_pi.Name);
 						}
 					} else {
 						if (field_pi.PropertyType == TypeManager.System_String)
 							throw new BindingException (1013, true, "Unsupported type for Fields (string), you probably meant NSString");
 						else
-							throw new BindingException (1014, true, "Unsupported type for Fields: {0}", fieldTypeName);
+							throw new BindingException (1014, true, "Unsupported type for Fields: {0} for '{1}'.", fieldTypeName, field_pi);
```

Comment:
why `field_pi` instead of `field_pi.Name` like the others ?

Reply:
Maybe use C#6 and a nicer message like:

```
throw new BindingException (1014, true, $"Unsupported type {fieldTypeName} for field '{field_pi.Name}'.");
```

Reply:
Because... mmm... I'll fix it 🙈